### PR TITLE
Created welcome Message yml

### PR DESCRIPTION
**Pull request by Alex-is-Gonzalez**
Created welcome message upon creating a first issue and first pr.

### CODE REVIEW

**Review by paramsiddharth — APPROVED** [review]
Looks good to me!

@JamesMGreene Do you think we'd be able to test this out using your second account, once merged?

File: .github/workflows/welcome_message.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+on:
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  welcome-new-contributor:
+    runs-on: ubuntu-latest
+    steps:
```
Comment (JamesMGreene) [inline .github/workflows/welcome_message.yml:12]: I recommend adding [a `timeout-minutes` configuration](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes) for **every** Actions job to ensure you don't end up wasting the default 6 hours in the case of a runaway Action/job:

```suggestion
    timeout-minutes: 2
    steps:
```

File: .github/workflows/welcome_message.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: 'Welcome New Contributors'
+
+on:
+  issues:
+    types: [opened]
+  pull_request_target:
+    types: [opened]
+
```
Comment (JamesMGreene) [inline .github/workflows/welcome_message.yml:8]: It's a best practice to restrict the permissions granted to the `GITHUB_TOKEN` to just the minimum set that are relevant to the workflow/job:

```suggestion

permissions:
  issues: write
  pull-requests: write

```

File: .github/workflows/welcome_message.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+jobs:
+  welcome-new-contributor:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Greet the contributor'
+        uses: garg3133/welcome-new-contributors@v1.2
```
Comment (JamesMGreene) [inline .github/workflows/welcome_message.yml:14]: From a security perspective, I would recommend you pin to a specific commit of the Action to ensure its tag (e.g. `v1.2`) is not maliciously swapped out from under you without giving you the opportunity to review it:

```suggestion
        uses: garg3133/welcome-new-contributors@a38583ed8282e23d63d7bf919ca2d9fb95300ca6
```

**Review by JamesMGreene — COMMENTED** [review]
This workflow looks good overall, though I left a few suggestions for best practices.

I don't see anything here that would warrant the need to run it in a separate account first, so just make sure to keep an eye out for potential failing runs in the future. 👍🏻

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
@Alex-is-Gonzalez I think it's best we move forward with James's advice and update the workflow. :)

**Comment by Alex-is-Gonzalez** [comment]
@JamesMGreene Thank you for the thorough feedback! I learned a lot from this, I went ahead and changed the yml based on the notes provided!

**Review by JamesMGreene — APPROVED** [review]
LGTM. Thanks for following up on my suggestions! ❤️

**Pull request merged**